Libraries/LibGUI/GTextEditor.h

```diff
@@ -257,6 +257,7 @@ class GTextEditor : public GScrollableWidget {
     bool m_automatic_indentation_enabled { false };
     bool m_line_wrapping_enabled { false };
     bool m_readonly { false };
+    int m_prev_y_offset;
```

Comment:
This is uninitialized so it won't have a valid value the first time we check it.
Also, it's very awkward that `m_prev_y_offset` is such a specific name that refers to a local variable name in `recompute_all_visual_lines()`.
Can't we instead compare this value to the current `content_size().height()`?
With the new way we compute the content height, it should be the same as `y_offset` here, no?
Or maybe I'm wrong, I'm kinda falling asleep here :)